checkstyle-suppressions.xml

```diff
@@ -19,5 +19,5 @@ limitations under the License.
 -->
 <suppressions>
   <!-- Suppress paragraph validation in copyright header -->
-  <suppress files=".*\.java" checks="JavadocParagraph" lines="1-12"/>
+  <suppress files=".*\.java" checks="JavadocParagraph" lines="1-16"/>
```

Comment:
yes, I deliberately added it so it would workaround a buggy checkstyle requirement for <p> tags, though if I add the copyright header below that's not going to work anymore. 

Adding the <p> tag that checkstyle wants makes the code less legible. The problem is that checkstyle does not recognize other non-p block-level elements such as pre.